suse_migration_services/units/mount_system.py

```diff
@@ -119,6 +123,16 @@ def initialize_logging():
         log.set_logfile(Defaults.get_migration_log_file())
 
 
+def inherit_ssh_host_keys():
+    system_ssh_host_keys_glob_path = Defaults.get_system_ssh_host_keys_glob_path()
+
+    if os.path.exists(os.path.dirname(system_ssh_host_keys_glob_path)):
+        system_ssh_host_keys = glob.glob(system_ssh_host_keys_glob_path)
+
+        for system_ssh_host_key in system_ssh_host_keys:
+            shutil.copy(system_ssh_host_key, '/etc/ssh/')
+
```

Comment:
The copy is not enough, sshd must be restarted after new host keys got added or changed

Reply:
In addition why is the ssh host key handling done in the mount_system service. This seems to be the wrong place. We have an explicit ssh_keys service. imho all of this belongs there not in mount_system

Reply:
I put it in mount_system so keys get copied as soon as possible.

Reply:
That's not needed as you need to restart sshd there is no hurry to do this. In addition nobody could login prior to setting up authorized_keys in the ssh-keys service. So from my perspective the ssh-keys service is the right place and it should do:

1. get the host keys 
2. restart sshd
3. get the pubkeys from the users and put in authorized keys (already implemented)

Reply:
So looks better now, thanks much

Reply:
Good. If all ok, I will squash them into one and merge.